package.json

```diff
@@ -22,7 +22,9 @@
     "graph:generate:staging": "yarn graph:generate staging",
     "graph:generate:development": "run(){ yarn graph:fetch:development && graphql-codegen --config codegen.ts && rm schema.graphql && yarn graph:format; }; run",
     "caddy:trust-ca": "bash ./scripts/trust-caddy-cert.sh",
-    "changelog": "yarn conventional-changelog -p conventionalcommits -i CHANGELOG.md -s"
+    "changelog": "yarn conventional-changelog -p conventionalcommits -i CHANGELOG.md -s",
+    "storybook": "storybook dev -p 6006",
+    "build-storybook": "storybook build"
```

Comment:
Bit of a nitpick: can we change these to ``storybook:dev` and `storybook:build` to keep a similar format as the other commands.

Reply:
I was wondering if I should do that or not too, thanks, will update it.